Documentation/ceph-cluster-crd.md

```diff
@@ -1023,3 +1000,41 @@ spec:
   cephVersion:
     image: ceph/ceph:v15.2.4 # Should match external cluster version
 ```
+
+### Cleanup policy
+
+Rook has the ability to cleanup resources and data that were deployed.
+The policy represents the confirmation that cluster data should be forcibly deleted.
+The cleanupPolicy should only be added to the cluster when the cluster is about to be deleted.
+After the `confirmation` field of the cleanup policy is set, Rook will stop configuring the cluster as if the cluster is about to be destroyed in order to prevent these settings from being deployed unintentionally.
+The `cleanupPolicy` CR settings has different fields:
+
+* `confirmation`: If `yes-really-destroy-data` the operator will automatically delete data on the hostpath of cluster nodes and clean devices with OSDs when a `delete cephcluster` command is issued. Only `yes-really-destroy-data` and an empty string are valid values for this field.
+* `sanitizeDisks`: sanitizeDisks represents advanced settings that can be used to sanitize drives.
+By default Rook, will only remove Ceph's metadata, which allows a cluster re-installation if desired.
+However, the administrator might want to sanitize the drives in more depth with the following flags:
+  * `method`: indicates if the entire disk should be sanitized or simply ceph's metadata. Possible choices are 'complete' or 'quick' (default)
+  * `dataSource`: indicate where to get random bytes from to write on the disk. Possible choices are 'zero' (default) or 'random'.
+  Using random sources will consume entropy from the system and will take much more time then the zero source
+  * `iteration`: overwrite N times instead of the default (1). Takes an integer value
+
+To automate activation of the cleanup, you can use the following command **AT YOUR OWN RISK**:
+
+```console
+kubectl -n rook-ceph patch cephcluster rook-ceph --type merge -p '{"spec":{"cleanupPolicy":{"confirmation":"yes-really-destroy-data"}}}'
+```
+
+When applied you will see the following from the Operator's logs:
```

Comment:
I don't think we need to show the detailed output from the log. I'd suggest removing lines 1027-1037

Reply:
I thought it was a useful indication but it might change so let's remove it. We already have a message about the cleanup policy being detected.